themes/default/content/docs/support/troubleshooting.md

```diff
@@ -105,9 +96,39 @@ To view a trace locally navigate to the [Jaeger UI](http://localhost:16686/searc
 
 ## Common Problems
 
-This section covers a few problems that can arise when working with Pulumi.
+### 403 error fetching plugin
+
+This error commonly occurs when using an arm64 based processor while using an older version of a provider that does not support arm64. It's not possible to upgrade the packages as your state will still be locked to the old version of the provider.
+
+The method for fiing this depends on whether you are using an Intel based computer.
+
+#### Intel based computer
+
+1. Open your Pulumi program on a non-arm64 based computer.
+1. Update your packages (pip / nuget / npm / go) and run `pulumi up`.
+1. Once the update is complete, open the new, updated Pulumi program on your arm64-based system.
+
+#### Non-Intel based computer
+
+1. Remove Pulumi - if you're using Homebrew, `brew remove pulumi` or `rm -rf ~/.pulumi`
```

Comment:
Aren't these separate things? `brew remove pulumi` would remove the Pulumi executable (`/opt/homebrew/bin/pulumi`), but `rm -rf ~/.pulumi` would remove plugins, templates, etc.